journal.py

```diff
@@ -105,13 +151,36 @@ class Entry(Base):
     created = sa.Column(
         sa.DateTime, nullable=False, default=datetime.datetime.utcnow)
 
+    def render_text(self):
+        return markdowner.convert(self.body_text)
```

Comment:
good!  Incorporating this as part of the API of the entry class makes good sense from an OO standpoint.